// Copyright (c) 2023 Boston Dynamics, Inc.  All rights reserved.
//
// Downloading, reproducing, distributing or otherwise using the SDK Software
// is subject to the terms and conditions of the Boston Dynamics Software
// Development Kit License (20191101-BDSDK-SL).

syntax = "proto3";

package bosdyn.api.spot;
option java_outer_classname = "DoorCommandServiceProto";

import "bosdyn/api/spot/door.proto";

service DoorService {
    rpc OpenDoor(OpenDoorCommandRequest) returns (OpenDoorCommandResponse) {}

    rpc OpenDoorFeedback(OpenDoorFeedbackRequest) returns (OpenDoorFeedbackResponse) {}
}

